ui/components/invitations/invitation-details.tsx

```diff
@@ -108,16 +109,15 @@ export const InvitationDetails = ({ attributes }: InvitationDetailsProps) => {
         </CardBody>
       </Card>
       <div className="flex w-full items-center justify-end">
-        <CustomButton
-          asLink="/invitations/"
+        <CustomLink
+          path={"/invitations/"}
```

Comment:
@sumit-tft can we have the same height that we have for the buttons?

![image](https://github.com/user-attachments/assets/0db0a92e-9c28-4978-9a23-c512c2c17507)